src/game/common/rts/player.h

```diff
@@ -57,7 +57,12 @@ struct SpecialPowerReadyTimerType // #TODO Implement and move elsewhere
 
 class AcademyStats // #TODO Implement and move elsewhere
 {
-    int members[45];
+public:
+    char unk[0x9C];
+    unsigned int mines_disarmed;
```

Comment:
m_minesCleared, based on string it uses (well should had used but someone made a copy paste mistake with ACADEMY:NoIncome) ACADEMY:ClearMines

Reply:
fixed.